tests/test_generate_auth_token.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import jwt
+
+from src.lab_orchestrator_lib_auth.auth import generate_auth_token, LabInstanceTokenParams
+
+
+class GenerateAuthTokenTestCase(unittest.TestCase):
+    def test_basic_token_generation(self):
+        # define token that is expected to be generated.
+        expected_token = b'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NSwiZXhwIjo4NjMzMjcyMDQ4LCJsYWJfaW5zdGFuY2UiOnsibGFiX2lkIjoxLCJsYWJfaW5zdGFuY2VfaWQiOjksIm5hbWVzcGFjZV9uYW1lIjoicGVudGVzdC11YnVudHUtMy05IiwiYWxsb3dlZF92bWlfbmFtZXMiOlsidWJ1bnR1Il19fQ.dRmdjiuKdFeyu2HHJWJQVspW1Aw1rMB6dJQe7LoLZww'
```

Comment:
use normal string instead of `b`-string

Reply:
yes, that was due to a wrong version installed on my machine

Reply:
fixed